### PR TITLE
auto format test_nestedtensor.py

### PR DESCRIPTION
**Pull request by YuqingJ**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #112456

### CODE REVIEW

**Comment by Skylion007** [comment]
Can we add this file to to the coverage of the UFMT in lintrunner?

File: .lintrunner.toml [hunk 1]
```diff
@@ -1544,7 +1544,6 @@ exclude_patterns = [
     'test/test_namedtuple_return_api.py',
     'test/test_native_functions.py',
     'test/test_native_mha.py',
-    'test/test_nestedtensor.py',
```
Comment (jbschlosser) [inline .lintrunner.toml:1547]: Given that other test files are skipped, why auto-format this one?

Comment (YuqingJ) [reply .lintrunner.toml:1547]: For this PR, I just want to auto format the whole file before I make other logic changes so they won't be mixed together. 
Should all tests ignored by UFMT linter? If that's the case, I can add it back to the exclude list.

**Review by davidberard98 — APPROVED** [review]
imo this is fine; looks like the tests are all excluded from ufmt just because they are grandfathered in from before ufmt was applied. seems like there's no harm in adding ufmt and suppressing vscode warnings

maybe give @jbschlosser a day in case there's other concerns?

**Pull request closed without merging**